week2/938_range_sum_bst.py

```diff
@@ -0,0 +1,28 @@
+# https://leetcode.com/problems/range-sum-of-bst/description/
+from typing import Optional
+
+
+class TreeNode:
+    def __init__(self, val=0, left=None, right=None):
+        self.val = val
+        self.left = left
+        self.right = right
+
+
+class Solution:
+    def range_sum_bst(self, root: Optional[TreeNode], low: int, high: int) -> int:
+        # using nested function
+        def sum_nodes(node):
+            current_node = node
```

Comment:
This variable seems unnecessary. Either rename `node` to `current_node` or please just remove this variable.